packages/grafana-ui/src/components/Table/TableNG/Cells/TableCellActions.tsx

```diff
@@ -69,7 +69,7 @@ export function TableCellActions(props: TableCellActionsProps) {
             }}
           />
           <IconButton
-            name={'search-minus'}
+            name={'filter-out-value'}
```

Comment:
Would this be a better name?
```suggestion
            name={'filter-plus'}
```